atom/browser/ui/certificate_trust_win.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2013 GitHub, Inc.
```

Comment:
Are we sure that there isn't a Chromium version of this code? https://cs.chromium.org/chromium/src/net/ssl/client_cert_store_win.cc?q=CertAddCertificateContextToStore+package:%5Echromium$&dr=C looks promising